tests/basics/class_contains.py

```diff
@@ -21,3 +21,24 @@ def __contains__(self, key):
 print(1 in b)
 print(2 in b)
 print(3 in b)
+
+
+class TestContains:
```

Comment:
Please call this class "C", to match the above tests.

Reply:
Renamed to C